### PR TITLE
Added Synpress with Auth test cases

### PR DESCRIPTION
**Pull request by jakzaizzat**
## Changes

Added `synpress` to handle e2e scenario with metamask and cover several test cases for basic authentication

1. User able to sign in with metamask and will redirect to his gallery
2. User able to sign out from his gallery

## How to run

1. Added `PRIVATE_KEY='xxx` and `NETWORK_NAME=main` to your environmental variable and run `yarn run synpress:run` 

or

2. Run `PRIVATE_KEY=xxx NETWORK_NAME=main  yarn run synpress:run`

Demo:
https://cln.sh/hx4ogv

### CODE REVIEW

**Comment by Robinnnnn** [comment]
hey, looks like this is the error!

image

File: tests/e2e/specs/home-spec.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    cy.url().should('include', '/members');
+  });
+
+  it('should redirect to collection page when click the sign in button', () => {
+    home.getSignInButton().should('be.exist');
+    home.getSignInButton().click();
+    home.getMetaMaskButton().click();
+    home.acceptMetamaskAccessRequest();
+    cy.confirmMetamaskSignatureRequest();
+    // TODO: Replace this with fixture
```
Comment (kaitoo1) [inline tests/e2e/specs/home-spec.ts:26]: what does this mean? is the TODO for this PR?

Comment (jakzaizzat) [reply tests/e2e/specs/home-spec.ts:26]: I'm thinking to address this on upcoming PR when we added more test cases. 

Basically, move all the hardcoded data into a fixture config - https://docs.cypress.io/api/commands/fixture#Examples

**Review by Robinnnnn — APPROVED** [review]
LGTM!!!

**Pull request merged**